golang/Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN curl -s -f -O https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz \
 
 COPY ./analyzer /work/analyzer
 WORKDIR /work/analyzer
-RUN GOBIN=/usr/local/bin go install -mod=vendor ./cmd/...
+RUN GOBIN=/usr/local/bin go install -mod=mod ./cmd/...
```

Comment:
I couldn't figure out why the builds failed w/o `-mod=mod`.
Only when I add this, the build passes...

Reply:
Wait, what's this.
```console
$ ls -al /usr/local/go115/src/vendor/golang.org/x/
total 24
drwxr-xr-x 6 root root 4096 Oct 14 19:16 .
drwxr-xr-x 3 root root 4096 Oct 14 19:16 ..
drwxr-xr-x 9 root root 4096 Oct 14 19:16 crypto
drwxr-xr-x 9 root root 4096 Oct 14 19:16 net
drwxr-xr-x 3 root root 4096 Oct 14 19:16 sys
drwxr-xr-x 5 root root 4096 Oct 14 19:16 text
```

Reply:
It was due to a bug in CircleCI built-in git client.
Addressed by installing git with apk.